eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -19,7 +19,7 @@ parameters:
   alpine317Container: mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.17
   centOSStream8Container: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8
   centOSStream9Container: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream9
-  fedora38Container: mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-38
```

Comment:
This image does not exist 🙂

You will need to add Fedora 39 Dockerfiles to dotnet-buildtools-prereqs-docker: https://github.com/dotnet/dotnet-buildtools-prereqs-docker/tree/main/src/fedora

Copying the Dockerfiles from 38 should work (we don't need the helix ones for source-build), then just make sure one of the Dockerfiles builds. You may have to update some package version numbers if they've changed between releases.

Reply:
You'll also need to add entries to fedora's [manifest.json](https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/main/src/fedora/manifest.json).

Reply:
Should be good to go now that https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/934 has been merged!